src/dflt-params/duplicates.case

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+desc: >
+  It is a Syntax Error if BoundNames of FormalParameters contains any duplicate
+  elements.
```

Comment:
nice one! If I'm not wrong, here is the ref: https://tc39.github.io/ecma262/#prod-ArrowFormalParameters